docs/src/fluxcalculator.md

```diff
@@ -3,7 +3,7 @@
 This modules contains abstract types and functions to calculate surface fluxes in the coupler, or to call flux calculating functions from the component models.
 
 Fluxes over a heterogeneous surface (e.g., from a gridpoint where atmospheric cell is overlying both land and ocean) can be handled in two different ways:
-1. **Combined fluxes** (called with `CombinedStateFluxes()`)
+1. **Combined fluxes** (called with `CombinedStateFluxesMOST()`)
```

Comment:
Why did you add the `MOST` suffix? It would be nice to document what that refers to

Reply:
This refers to the Monin Obukhov Similarity Theory (the acronym is common in the field and is used in Atmos). I can clarify this in the comments though. :)

Reply:
Thank you!